app/components/curate-work/component.js

```diff
@@ -40,6 +40,21 @@ export default Ember.Component.extend({
         }];
     }.property('toMerge.[]'),
 
+    validLinks: Ember.computed('work.links', function() {
+        let links = this.get('work.links').content.currentState;
```

Comment:
Ember *should* take care of this for you. Using `this.get('work.links')` should be sufficient.